config/profile/rhdh/default-config/pvcs.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: audit-log
+  annotations:
+    "rhdh.redhat.com/mount-path": "/var/log/redhat-developer-hub/audit"
```

Comment:
is `"/var/log/redhat-developer-hub/audit"` right path? In helm chart this is mounted at `/var/log/audit`

Reply:
I have a pull request open to fix the helm chart one. https://github.com/redhat-developer/rhdh-chart/pull/45